docs/source/torch.rst

```diff
@@ -183,216 +223,240 @@ Examples::
   >>> y.requires_grad
   False
 
-.. autofunction:: no_grad
-.. autofunction:: enable_grad
-.. autofunction:: set_grad_enabled
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    no_grad
+    enable_grad
+    set_grad_enabled
 
 Math operations
 ----------------------------------
 
-Pointwise Ops
-~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: abs
-.. autofunction:: absolute
-.. autofunction:: acos
-.. autofunction:: add
-.. autofunction:: addcdiv
-.. autofunction:: addcmul
-.. autofunction:: angle
-.. autofunction:: asin
-.. autofunction:: atan
-.. autofunction:: atan2
-.. autofunction:: bitwise_not
-.. autofunction:: bitwise_and
-.. autofunction:: bitwise_or
-.. autofunction:: bitwise_xor
-.. autofunction:: ceil
-.. autofunction:: clamp
-.. autofunction:: conj
-.. autofunction:: cos
-.. autofunction:: cosh
-.. autofunction:: div
-.. autofunction:: digamma
-.. autofunction:: erf
-.. autofunction:: erfc
-.. autofunction:: erfinv
-.. autofunction:: exp
-.. autofunction:: expm1
-.. autofunction:: floor
-.. autofunction:: floor_divide
-.. autofunction:: fmod
-.. autofunction:: frac
-.. autofunction:: imag
-.. autofunction:: lerp
-.. autofunction:: lgamma
-.. autofunction:: log
-.. autofunction:: log10
-.. autofunction:: log1p
-.. autofunction:: log2
-.. autofunction:: logical_and
-.. autofunction:: logical_not
-.. autofunction:: logical_or
-.. autofunction:: logical_xor
-.. autofunction:: mul
-.. autofunction:: mvlgamma
-.. autofunction:: neg
-.. autofunction:: polygamma
-.. autofunction:: pow
-.. autofunction:: real
-.. autofunction:: reciprocal
-.. autofunction:: remainder
-.. autofunction:: round
-.. autofunction:: rsqrt
-.. autofunction:: sigmoid
-.. autofunction:: sign
-.. autofunction:: sin
-.. autofunction:: sinh
-.. autofunction:: sqrt
-.. autofunction:: square
-.. autofunction:: tan
-.. autofunction:: tanh
-.. autofunction:: true_divide
-.. autofunction:: trunc
-
+.. autosummary::
+    :toctree: generated/point
```

Comment:
Maybe you want to use `generated`?

Reply:
yes, fixed

Reply:
Not sure if it's just me, but it seems that this change doesn't fix the link issue mentioned above: `torch.mul` still links to the [torch.html#torch.mul](https://shawnzhong.github.io/pytorch/torch.html#torch.mul).

Reply:
Strange. I don't see this on two different builds (both with cuda installed, one on a machine with a GPU, one without). There is nothing obvious in `torch/_torch_docs.py` in the docstrings for these functions as overloaded from `add_docstr(torch.mul,...` and friends. What do you mean by "some overloaded functions" - is there something special about `mul` and `pow` that my builds are not picking up?

How are you building and uploading the docs to https://shawnzhong.github.io/pytorch ?

I guess I could try to replicate the actual `python_doc_push_script.sh` in a docker, does that CI step run with a GPU?

Reply:
I use a [GitHub action](https://github.com/ShawnZhong/pytorch/blob/e9e2d6a863f06234c8abd6a1796decefcbb6aafd/.github/workflows/sphinx.yml) to build and push the HTML files to `gh_pages`, which is served by GitHub pages. 

It might be a different environment that causes this issue. I tried to use the same environment as CircileCI, but it seems that I don't have the permission to pull the docker image for building docs. BTW, I just opened a PR to upload the build artifact for python docs at https://github.com/pytorch/pytorch/pull/37658.

Reply:
Nvm, it works on the CircleCI build. https://5319560-65600975-gh.circle-artifacts.com/0/~/workspace/build_artifacts/master/index.html